proposals/utils/proposal_utils.py

```diff
@@ -97,12 +97,14 @@ def available_urls(proposal):
         urls.append(studies_url)
 
 
-        consent_url = AvailableURL(title=_('Documenten'))
+        consent_docs_url = AvailableURL(title=_('Uploaden'), url=reverse('proposals:consent', args=(proposal.pk,)))
+        consent_url = AvailableURL(title=_('Formulieren'), children=[consent_docs_url])
+
         data_management_url = AvailableURL(title=_('Datamanagement'))
         submit_url = AvailableURL(title=_('Versturen'))
 
         if proposal.last_study() and proposal.last_study().is_completed():
-            consent_url.url = reverse('proposals:consent', args=(proposal.pk,))
+            consent_url.url = reverse('proposals:translated', args=(proposal.pk,))
```

Comment:
I did leave line 107 be, because the consent_docs_url is a child of the consent docs_url, so these lead to purposely different urls. I did implement all you other suggestions though.